CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+. Removed support for versions of Lightning lower than v1.5 ([#788](https://github.com/PyTorchLightning/metrics/pull/788))
```

Comment:
Is the description correct? It mentions 1.3 but in the code it uses 1.5

Reply:
Sorry forgot that to update the PR description (will do).
In PR #739 we increased from 1.0 to 1.3.
With this PR we are increasing from 1.3 to 1.5.

Reply:
Is #739 included in a different release than this PR?
Or is the next release going to jump 1.0 -> 1.5?

I'm fine either way (you can resolve this conversation), just curious.

@Borda You mentioned in the past that we should have a compatibility matrix table for our different projects, I think it's a great idea, and doing it before the 1.6 release will help `torchmetrics` users.

Reply:
#739 was in 0.7 so jump will be from 1.3 -> 1.5 :]

Reply:
We can do PL docs or README to begin with and if there's a better place later, we move it

Other projects can link to it.

Reply:
Created issue on lightning side for future discussion: https://github.com/PyTorchLightning/pytorch-lightning/issues/11717